portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -47,15 +47,39 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+<<<<<<< HEAD
```

Comment:
There are diff errors here. This doesn't compile.

Reply:
That was made default by github